tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -54,19 +54,27 @@ function () {
 			}
 		);
 
-		$spy = new MethodSpy();
-		add_action( 'http_api_debug', array( $spy, 'callback' ), 10, 5 );
+		$pre_args = null;
+		$pre_url  = null;
+
+		// Use pre_http_request to support WP < 5.3
```

Comment:
Please elaborate a little more in this comment as to why this is necessary.
Specifically because `http_api_debug` is not fired for blocked requests before WP 5.3 😄

Reply:
done